.github/workflows/commit.yaml

```diff
@@ -46,10 +46,10 @@ jobs:
         run: make check
 
       - name: "Run unit tests"
-        # prefetch implicit version needed by pkg/binary/envoy/controlplane/istio_test.go until #136. This avoids:
-        # Unable to start Envoy process: fork/exec /home/circleci/.getenvoy/builds/standard/1.11.0/linux_glibc/bin/envoy: text file busy
+        # Prefetch envoy to avoid latency during "go test" which contributes to timeouts.
+        # This version must align with pkg/binary/envoytest/util.go
         run: |
-          go run cmd/getenvoy/main.go fetch standard:1.11.0
+          go run cmd/getenvoy/main.go fetch standard:1.12.7
```

Comment:
Let's update this to latest, 1.17.1?

Reply:
There are some issues in doing that related to dropped config v2 support, so want to make a step first. Happy to do that later according to #169 sg?

Reply:
yeah let's do v3 support and upgrading to 1.17.1 in a separate PR